ledger/participant-integration-api/src/main/scala/platform/apiserver/LedgerFeatures.scala

```diff
@@ -13,4 +15,6 @@ case class LedgerFeatures(
     commandDeduplicationFeatures: CommandDeduplicationFeatures =
       CommandDeduplicationFeatures.defaultInstance,
     contractIdFeatures: ExperimentalContractIds = ExperimentalContractIds.defaultInstance,
+    committerEventLog: ExperimentalCommitterEventLog =
```

Comment:
Nit: You could simply use `ExperimentalCommitterEventLog.defaultInstance` here for consistency.